tools/webtransport/h3/webtransport_h3_server.py

```diff
@@ -551,7 +578,16 @@ def _start_on_server_thread(self) -> None:
         _logger.info("Starting WebTransport over HTTP/3 server on %s:%s",
                      self.host, self.port)
 
-        configuration.load_cert_chain(self.cert_path, self.key_path)
+        if self.cert_mode == WebTransportCertificateGeneration.USEPREGENERATED:
+            configuration.load_cert_chain(self.cert_path, self.key_path)
+        else: # GENERATEDVALIDSERVERCERTIFICATEHASHCERT case
```

Comment:
Let's add an assert or something there, verify that we are actually this case (and haven't missed this if/when we add another new `cert_mode`).

Reply:
done!